packages/widget_toolkit/lib/src/lib_edit_address/ui_components/edit_address_widget.dart

```diff
@@ -48,6 +48,7 @@ class EditAddressWidget<T extends PickerItemModel> extends StatefulWidget {
     required this.translateError,
     required this.service,
     this.onChanged,
+    this.onSaved,
```

Comment:
It would be nice to have a comment above the class with information about `onSaved`

Reply:
And also in the CHANGELOG.md

Reply:
Done